pkg/component/etcd/service/service_client.go

```diff
@@ -25,12 +25,12 @@ import (
 
 func (c *component) syncClientService(ctx context.Context, svc *corev1.Service) error {
 	_, err := controllerutils.GetAndCreateOrStrategicMergePatch(ctx, c.client, svc, func() error {
-		svc.Labels = utils.MergeStringMaps(getLabels(c.values), c.values.ClientServiceLabels)
+		svc.Labels = utils.MergeStringMaps(getLabels(c.values), c.values.ClientServiceLabels, getSelectors(c.values))
```

Comment:
This seems unnecessary to me. We should just have a default set of labels for a client service and then add additional labels that are specified in the etcd CRD via `spec.clientService.Labels`

Reply:
I did this for uniformity between service and statefulsets. I think, if it's not harming, we can keep it

Reply:
network* specific labels are not really needed on a service. Having labels which were never meant for a resource is quite misleading. This was also a comment in the slack channel and i agree with it.

Reply:
Can it be part of the refactoring work that you are doing? Let it go through for now. Let the gardener upgrade tests pass and then we can discuss what can be the common labels for services